### ISSUE TITLE
Bodies vibration

### ISSUE THREAD

**Issue by NIRumiantsev** [pmndrs/cannon-es#129]
I currently learn cannon.js and struggle with the strange problem. All physical bodies produce some kind of vibrations when they stacked to each other. You can see it even in your own example: https://pmndrs.github.io/cannon-es/examples/split_solver .
The frequency depends on solver iterations and gravitation. Anyone faced this issue before?

**Comment by marcofugaro** [comment]
Can you record a video? I am not seeing it..

**Comment by marcofugaro** [comment]
Does it happen for the jenga example as well? https://pmndrs.github.io/cannon-es/examples/jenga

**Comment by NIRumiantsev** [comment]
> Does it happen for the jenga example as well? https://pmndrs.github.io/cannon-es/examples/jenga

Yes it does! You can just set solver iterations to 1 and you will see jenga crashes
